chart/templates/configmap-env.yaml

```diff
@@ -12,7 +12,11 @@ data:
   {{- end }}
   DB_NAME: {{ .Values.postgresql.auth.database }}
   DB_POOL: {{ .Values.mastodon.sidekiq.concurrency | quote }}
+  {{- if .Values.postgresql.auth.port }}
+  DB_PORT: {{ .Values.postgresql.auth.port }}
```

Comment:
nitpicking: but this could use the [default function](https://helm.sh/docs/chart_template_guide/functions_and_pipelines/#using-the-default-function) instead

Reply:
thanks